fraud/nmt.go

```diff
@@ -0,0 +1,79 @@
+// TEMPORARY
```

Comment:
Remind me why we have to duplicate this code? Can't we just use the implementation in core? Because that Tree() method that was added but there was no tagged release yet?

Reply:
We can't depend on the latest core rn due to app not being updated with PreProcessProposal or something not implemented

Reply:
I also think we should quickly do a release and depend on that before we merge this PR. Should we set it back to draft and mention the blocker in the opening comment? 
Alternative: continue reviewing and merge this with the code duplication but then we should open an issue to remove this before we merge the PR.

Reply:
see: #607

Reply:
File is removed